Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.cpp

```diff
@@ -340,6 +370,7 @@ namespace AzFramework
             for (auto it = ticket.m_spawnedEntities.begin() + spawnedEntitiesInitialCount; it != ticket.m_spawnedEntities.end(); ++it)
             {
                 GameEntityContextRequestBus::Broadcast(&GameEntityContextRequestBus::Events::AddGameEntity, *it);
+                (*it)->SetSpawnTicketId(request.m_ticketId);
```

Comment:
Repeated: It's safer to set the id first and then broadcast to instance it in the world. It's theoretically possible that during the instancing the entity is immediately destroyed.